plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/translator/DAIHistoryItem.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.model.ai.translator;
+
+import org.jkiss.dbeaver.model.data.json.JSONUtils;
+
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Natural language translator item
+ */
+public class DAIHistoryItem {
+
+    private String id;
+    private String naturalText;
+    private String completionText;
+    private Date time;
+
+    public DAIHistoryItem() {
+    }
+
+    public DAIHistoryItem(String naturalText, String completionText) {
+        this.naturalText = naturalText;
+        this.completionText = completionText;
+    }
+
+    public DAIHistoryItem(Map<String, Object> map) {
+        this.id = JSONUtils.getString(map, "id");
+        this.naturalText = JSONUtils.getString(map, "naturalText");
+        this.completionText = JSONUtils.getString(map, "completionText");
+        this.time = new Date(JSONUtils.getLong(map, "time", 0));
```

Comment:
These strings are used at least twice. I'd make them to be constants.

Reply:
We want replace them with native json serializatiom. It is temporary